core/src/ephemeral_server/mod.rs

```diff
@@ -326,6 +233,12 @@ impl EphemeralServer {
             Ok(())
         }
     }
+
+    /// Get the process ID of the child. This will be None if the process is
+    /// considered to be complete.
```

Comment:
> considered to be complete

Note, this is vague on purpose, ref https://docs.rs/tokio/latest/tokio/process/index.html#unix-processes